manifests/database.pp

```diff
@@ -13,16 +13,18 @@
     user     => $gutterball::dbuser,
     password => $gutterball::dbpassword,
   } ~>
+  file { '/usr/bin/gbdb':
```

Comment:
This may be a nitpick, but maybe make it explicitly ```gutterball-db```? This is very close to gdb.